TrackingTools/TrackRefitter/src/TrackTransformer.cc

```diff
@@ -113,6 +138,10 @@ TrackTransformer::getTransientRecHits(const reco::TransientTrack& track) const {
 	  continue;
 	}
 	result.push_back(theMuonRecHitBuilder->build(&**hit));
+      } else if ( (*hit)->geographicalId().det() == DetId::Forward && 
+		  (*hit)->geographicalId().subdetId() == FastTime  ) {
+	if (  theMtdAvailable ) result.push_back(theMTDRecHitBuilder->build(&**hit));
```

Comment:
@Martin-Grunewald this check should ensure backward consistency of results when MTD is not present